flocker/node/agents/test/test_blockdevice.py

```diff
@@ -169,12 +169,10 @@
 
 DESIRED_DATASET_ATTRIBUTE_STRATEGIES = {
     'dataset_id': uuids(),
-    'maximum_size': integers(min_value=0).map(
-        lambda n: (
-            LOOPBACK_MINIMUM_ALLOCATABLE_SIZE +
-            n * LOOPBACK_ALLOCATION_UNIT
-        )
-    ),
+    'maximum_size': integers(
```

Comment:
❓ I think the other formation is (a  lot) better for hypothesis:

```
integers(min_value=0, max_value = 10).map(lambda n: LOOPBACK_MINIMUM_ALLOCATABLE_SIZE+n*LOOPBACK_ALLOCATION_UNIT)
```

Otherwise I think hypothesis is like "try 512... okay now try 576... okay now try 513... okay now try 514" which is a whole bunch of wasted examples (as they would all be run as 512) (and real test time) that cheapens hypothesis's ability to fuzz the relevant part of the test input search space.

Apologies I don't have time to hack together a minimal example to show you right now... but I'm happy to make one if you need assistance.